pbkhrv/rtl_433-hass-addons
# Disable TPMS sensors by default #110

## Summary

This PR updates the template to disable TPMS sensors by default. The list of protocols is from `master` in rtl_433.

Fixes #110

## Alternatives Considered

We could add some sort of exclusion from the autodiscovery script, but this will both reduce the amount of MQTT traffic and doesn't require writing any new code.

## Testing Steps

1. Move any existing configuration files out of the way.
2. Start or restart the rtl_433 addon.
3. TPMS sensors should not be logged.
